templates/rhdm710-authoring-ha.yaml

```diff
@@ -720,7 +720,11 @@ objects:
           - name: SERVICE_PROFILE
             value: ${APPLICATION_NAME}-datagrid
           - name: JGROUPS_PING_PROTOCOL
-            value: openshift.DNS_PING
+            value: kubernetes.KUBE_PING
```

Comment:
and about the other deployment configs?
plus, should't the ping service and port be removed from  kieserver and bc?

same applies for the other templates and rhpam as well.